.github/workflows/docker-hub.yaml

```diff
@@ -25,10 +21,10 @@ jobs:
         id: meta
         uses: docker/metadata-action@v5
         with:
-          flavor: latest=false
+          flavor: latest=true
           images: smartonfhir/cumulus-etl
           tags: |
-            type=ref,event=branch
+            type=pep440,pattern={{major}},value=${{ env.ETL_VERSION }}
```

Comment:
This drops the docker build for every single commit (the `:main` tag). I could have kept it, but it felt less useful now that we have real releases... Do you like the idea of keeping them?

Reply:
Yeah, if you're commiting to versions, the docker versions should match the pypi version.